cupy/core/raw.pxd

```diff
@@ -7,6 +7,7 @@ cdef class RawKernel:
         readonly tuple options
         readonly str backend
         readonly bint enable_cooperative_groups
+        tuple specializations
```

Comment:
Could you rename to `_specializations` if this attribute is only for internal implementation?

Reply:
I think it's already because it's a `cdef` attribute. It's not visible in the Python space, so let me save one key stroke 😛